lib/str-repr/tests/test_encode.c

```diff
@@ -75,21 +53,19 @@ assert_encode_with_forbidden_equals(const gchar *input, const gchar *forbidden_c
   GString *str = g_string_new("");
 
   str_repr_encode(str, input, -1, forbidden_chars);
-  assert_string(str->str, expected, "Encoded value does not match expected");
+  cr_assert_str_eq(str->str, expected, "Encoded value does not match expected");
```

Comment:
The Criterion assertion API is a bit different than the old one. If you specify the assertion description string, the actual and expected values will not be shown.
Could you please add them back?

For example:
```
cr_assert_str_eq(str->str, expected, "Encoded value does not match expected; actual: %s, expected: %s", str->str, expected);
```

Reply:
ok. I will change it..